src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/.template.config/ide.host.json

```diff
@@ -14,6 +14,12 @@
       "isVisible": true,
       "persistenceScope": "templateGroup"
     },
+    {
+      "id": "auth",
+      "isVisible": true,
+      "defaultValue": "None",
+      "persistenceScope": "templateGroup"
```

Comment:
I'm not sure if `"persistenceScope": "templateGroup"` is correct or if `ide.host.json` even supports enumerating choices (i.e. "None" and "Individual"). I only see boolean options in the other project templates.

None of the other current project templates do not expose the "auth" option via the `File > New > Project` dialog in VS presumably because it can normally be scaffolded in later if you're using VS.